rootfs/scheduler/__init__.py

```diff
@@ -1371,6 +1371,19 @@ def _pod_liveness_status(self, pod):
 
         return True
 
+    def _pod_ready(self, pod):
+        """Combines various checks to see if the pod is considered up or not by checking probes"""
+        if (
```

Comment:
This could be just a boolean expression.
```python
return (
    pod['status']['phase'] == 'Running' and
    # is the readiness probe passing?
    self._pod_readiness_status(pod) == 'Running' and
    # is the pod ready to serve requests?
    self._pod_liveness_status(pod)
)
```